apps/docs-snippets/src/guide/introduction/getting-started.test.ts

```diff
@@ -1,22 +1,36 @@
-import { FUEL_NETWORK_URL, TESTNET_NETWORK_URL, Provider, Wallet, WalletUnlocked } from 'fuels';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-shadow */
+import { TESTNET_NETWORK_URL, Provider, Wallet, WalletUnlocked } from 'fuels';
+import { launchTestNode } from 'fuels/test-utils';
 
 /**
  * @group node
  * @group browser
  */
 describe('Getting started', () => {
-  beforeAll(async () => {
-    // Avoids using the actual network.
-    const mockProvider = await Provider.create(FUEL_NETWORK_URL);
-    vi.spyOn(Provider, 'create').mockResolvedValue(mockProvider);
+  let destroy: () => void;
+  let url: string;
+
+  beforeEach(async () => {
+    const { cleanup, provider } = await launchTestNode({
+      nodeOptions: {
+        port: '4000',
+      },
+    });
+    destroy = cleanup;
+    url = provider.url;
+  });
+
+  afterEach(() => {
+    destroy();
```

Comment:
Could we replace this chunk with the standard approach inside the test in question?

```ts
// eslint-disable-next-line @typescript-eslint/no-unused-vars
using launched = await launchTestNode({
  nodeOptions: { port: '4000' },
});
```

Reply:
283c96e33